django2pydantic/__init__.py

```
@@ -1,19 +1,24 @@
-"""Super schema packages."""
+"""\ndjango2pydantic is a library that allows you to define Pydantic schemas based on Django database models.\n\nThis module exposes the public API for the django2pydantic library, including schema definition classes, field type registry, and utility types for inferring schema details from Django models.\n"""
```

Ruff (0.8.2)

Use `r"""` if any backslashes in a docstring

Add `r` prefix

(D301)